.github/workflows/pytest.yml

```diff
@@ -24,16 +24,16 @@ jobs:
         with:
           repository: "oceanprotocol/barge"
           path: 'barge'
-          ref: v4
+          ref: feature/v4_c2d
       - name: Run Barge
         working-directory: ${{ github.workspace }}/barge
         env:
```

Comment:
please remove this, because it's using the remote deployment and not barge's

Reply:
Done.